lib/api_repo.js

```diff
@@ -1018,11 +1021,14 @@ ApiRepo.prototype._makeProtocFunc = function _makeProtocFunc(opts, language) {
           pluginOption = '--plugin=';
         } else {
           args.push('--' + language + '_out=' + outDir);
-          if (language === 'python') {
-            // required per https://github.com/grpc/grpc/issues/7857
-            args.push('--grpc_python_out=' + outDir);
-          } else {
-            args.push('--grpc_out=' + outDir);
+          if (!opts.buildCommonProtos ||
+              _.includes(COMMON_GRPC_PROTOS, protoPath)) {
```

Comment:
isn't this the path from the proto's rootdir (i.e. 'google/longrunning/operations.proto')?